onnx_caffe2/backend.py

```diff
@@ -597,18 +680,20 @@ def prepare(cls, model, device='CPU', **kwargs):
 
     @classmethod
     # TODO: This method needs a refactor for clarity
-    def _onnx_node_to_caffe2_op(cls, node_def, opset_version):
+    def _onnx_node_to_caffe2_op(cls, init_model, pred_model, node_def, opset_version):
         if node_def.op_type in cls._special_operators:
             translator = getattr(cls, cls._special_operators[node_def.op_type])
         else:
             translator = cls._common_onnx_node_to_caffe2_op
-        ops = translator(OnnxNode(node_def), opset_version)
+        ops = translator(init_model, pred_model, OnnxNode(node_def), opset_version)
+        if isinstance(ops, tuple):
```

Comment:
maybe create a`namedtuple` for the return type? It'd be a bit safer than just returning 3 values.